markdownlint.js

```diff
@@ -131,30 +131,47 @@ function prepareFileList(files, fileExtensions, previousResults) {
 }
 
 function printResult(lintResult) {
+  let lintResultString = '';
```

Comment:
It doesn't look like this needed to move? Not a problem, just curious why?

Reply:
Yeah I moved it in an earlier iteration when I tried to refactor a bit. No real value now to move it.